pkg/sdn/plugin/node.go

```diff
@@ -272,11 +272,13 @@ func (node *OsdnNode) Start() error {
 				continue
 			}
 			if vnid, err := node.policy.GetVNID(p.Namespace); err == nil {
-				node.policy.RefVNID(vnid)
+				node.policy.EnsureVNIDRules(vnid)
 			}
 		}
 	}
 
+	go kwait.Forever(node.syncVNIDRules, 24*time.Hour)
```

Comment:
What happens if I create a pod with ip 10.1.11.17 in namespace A and VNID 2000. I then delete the pod. After that I start making pods in namespace B and VNID 3000 until the ip 10.1.11.17 is reused.
As I understand it at that momenent there are 2 sets of rules in the system. 
one for  10.1.11.17 in VNID 2000 and one for 10.1.11.17 in VNID 3000. How does that affect packets from or to address 10.1.11.17. Does it break the boundries between namespace A and B?